ansible_collections/arista/avd/plugins/plugin_utils/schema/avd_meta_schema.json

```diff
@@ -309,6 +314,39 @@
                 }
             },
             "additionalProperties": false
+        },
+        "deprecation": {
+            "type": "object",
+            "description": "Deprecation settings",
+            "required": ["warning"],
+            "properties": {
+                "warning": {
+                    "type": "boolean",
+                    "description": "Emit deprecation warning if key is set",
+                    "default": true
+                },
+                "new_key": {
+                    "type": "string",
+                    "description": "Relative path to new key"
+                },
+                "removed": {
+                    "type": "boolean",
+                    "description": "Support for this key has been removed",
+                    "default": false
+                },
+                "remove_in_version": {
+                    "type": "string",
+                    "description": "Version in which the key will be removed"
+                },
+                "remove_after_date": {
+                    "type": "string",
+                    "description": "Date after which the key will be removed"
```

Comment:
if we keep it probably worth stating that it is the date after which the key will be removed `in the next major version` (as this is what the error message says in python)

Reply:
fixed